index.html

```diff
@@ -1,29 +1,21 @@
 ﻿<!DOCTYPE html>
-<html lang="en">
-	<head>
-		<meta charset="UTF-8" />
-		<meta http-equiv="X-UA-Compatible" content="IE=Edge" />
-		<title>custom-functions-preview</title>
-		<script src="https://unpkg.com/@microsoft/office-js@custom-functions-preview-refresh/dist/custom-functions-runtime.js." type="text/javascript"></script>
-		<script src="src/customfunctions.js" type="text/javascript"></script>
-		<script type="text/javascript">
-			Office.Preview.startCustomFunctions();
-		</script>
-	</head>
-	<body>
-		Thanks for trying custom functions for Excel! You've installed the sample functions, which start with the prefix CONTOSO.
-		Try typing them into a cell (with or without this task pane - it isn't needed):
-		<ul>
-			<li><b>=CONTOSO.ADD42</b> adds 42 to two input numbers.</li>
-			<li><b>=CONTOSO.ADD42ASYNC</b> does the same thing but waits 1 second first.</li>
-			<li><b>=CONTOSO.ISPRIME</b> checks whether the number is prime and returns true or false.</li>
-			<li><b>=CONTOSO.NTHPRIME</b> returns the Nth prime. Watch out - it needs a lot of calculation if N is big!</li>
-			<li><b>=CONTOSO.GETDAY</b> returns the current day of the week as a string.</li>
-			<li><b>=CONTOSO.SECONDHIGHEST</b> lets you select a range of data and finds the second highest number.</li>
-			<li><b>=CONTOSO.INCREMENTVALUE</b> increments the value by the number specified.</li>
-		</ul>
-		To change the functions, edit <b>src/customfunctions.js</b> and <b>config/customfunctions.json</b>. To change the prefix, edit <b>manifest.xml</b>.
-		For more information, see <a href=https://aka.ms/customfunctions>https://aka.ms/customfunctions</a>.
-	</body>
-</html>
+<html>
 
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=Edge" />
+  <meta http-equiv="Expires" content="0" />
+  <title></title>
+  <script src="https://unpkg.com/@microsoft/office-js@custom-functions-preview-refresh/dist/custom-functions-runtime.js" type="text/javascript"></script>
+  <script src="dist/win32/ship/index.win32.bundle" type="text/javascript"></script>
+  <script type="text/javascript">
+    Office.Preview.startCustomFunctions();
+  </script>
+</head>
+
+<body>
+  Thanks for trying custom functions for Excel! For more information, see
```

Comment:
Can we also provide a list of the built in functions in this? I like that the taskpane did that previously and I think it could be helpful for users.